engine/class_modules/sc_death_knight.cpp

```diff
@@ -4390,49 +4397,16 @@ struct defile_damage_t : public death_and_decay_damage_base_t
   }
 };
 
-struct deaths_due_damage_t : public death_and_decay_damage_base_t
+struct deaths_due_damage_t : public death_and_decay_damage_t
 {
   int pestilence_procs_per_tick;
   int pestilence_procs_per_cast;
 
   deaths_due_damage_t( death_knight_t* p ) :
-    death_and_decay_damage_base_t( "deaths_due_damage", p, p -> find_spell( 341340 ) ),
+    death_and_decay_damage_t( "deaths_due_damage", p, p -> find_spell( 341340 ) ),
     pestilence_procs_per_tick( 0 ),
     pestilence_procs_per_cast( 0 )
   { }
-
-  double composite_da_multiplier( const action_state_t* state ) const override
-  {
-    double m = death_knight_spell_t::composite_da_multiplier( state );
-
-    m *= 1.0 + p() -> conduits.withering_ground.percent();
-
-    return m;
-  }
```

Comment:
!

Reply:
(You can also remove the pestilence related variables, it should all be handled by ``death_and_decay_damage_t`` now)